python_frank_energie/models.py

```
@@ -1,16 +1,28 @@
-"""Data models for the Frank Energie API."""
-
-from __future__ import annotations
+"""Data models enable parsing and processing of the Frank Energie API responses in a structured manner."""
+# python_frank_energie/models.py
 
 import logging
-from dataclasses import dataclass
-from datetime import datetime, timedelta, timezone
-
+import statistics
```

Ruff

`statistics` imported but unused

Remove unused import: `statistics`

(F401)